packages/react/src/primitives/TextField/TextField.tsx

```diff
@@ -49,6 +50,12 @@ const TextFieldPrimitive = <Multiline extends boolean>(
   const { flexContainerStyleProps, baseStyleProps, rest } =
     splitPrimitiveProps(_rest);
 
+  useDeprecationWarning({
+    shouldWarn: props.isMultiline === true,
```

Comment:
```suggestion
    shouldWarn: props.isMultiline,
```

`isMultiline` is a boolean itself :)